docs/getting-started.md

```diff
@@ -0,0 +1,130 @@
+    "build_date" : "2017-12-04T07:50:10.466Z",
+    "build_snapshot" : false,
+    "lucene_version" : "6.6.1"
+  },
+  "tagline" : "You Know, for Search"
+}
+```
+
+Verify that Couchbase Server is running.
+
+```bash
+$ curl localhost:8092
+
+{"couchdb":"Welcome","version":"v4.5.1-60-g3cf258d","couchbase":"5.0.2-5506-community"}
+```
+
+## Installation
+
+Navigate to the Elasticsearch installation directory.
+
+```bash
+$ cd /usr/share/elasticsearch
+```
+
+Download and install the plug-in package.
+
+```bash
+$ bin/elasticsearch-plugin install https://github.com/couchbaselabs/elasticsearch-transport-couchbase/releases/download/3.0.0-cypress/elasticsearch-transport-couchbase-3.0.0-cypress-es5.6.4.zip
+```
+
+Replace the plugin URL with the one that matches your Elasticsearch version, all URLs can be found on the [releases page](https://github.com/couchbaselabs/elasticsearch-transport-couchbase/releases).
+
+> **Note:** If you're using Elasticsearch 2.x, the command to install plugins is `bin/plugin` instead of `bin/elastichsearch-plugin`.
+
+When the installation is successful, the message "Installed transport-couchbase into /usr/share/elasticsearch/plugins/transport/couchbase" will be logged.
+
+## Configuration
+
+Open the Elasticsearch configuration file (**/etc/elasticsearch/elasticsearch.yml**) and add the following to the end of the file.
+
+```yaml
+couchbase.username: <USERNAME>
+couchbase.password: <PASSWORD>
+couchbase.maxConcurrentRequests: 1024
+```
+
+The **username** and **password** credentials will be used again later when configuring Couchbase Server.
+
+Still in the **/usr/share/elasticsearch** directory, configure the Elasticsearch plugin with the following curl command.
+
+```bash
+$ curl -X PUT http://localhost:9200/_template/couchbase -d @plugins/transport-couchbase/couchbase_template.json
+```
+
+When successful, the configuration-routine provides the following response: `{"acknowledged":true}`.
+
+Create an Elasticsearch index to receive the data from the Couchbase bucket. Later, when configuring XDCR, the "remote bucket name" should match the Elasticsearch **index name**.
+
+```bash
+$ curl -X PUT http://localhost:9200/travel-sample
+
+{"acknowledged":true}
+```
+
+Now to configure Couchbase Server, open the Couchbase Web Console and select **XDCR > Add Remote Cluster**.
+
+![](assets/xdcrInitial.png)
+
+In the dialog, enter the **Cluster Name** of your choice, the **IP/hostname** and port number where the Elasticsearch cluster is running (the Elasticsearch plugin listens on port 9091 by default) and the **Username**/**Password** previously stored in **elasticsearch.yml**.
+
+![](assets/remotecluster.png)
+
+Next, select the **Add Replication** option.
+
+![](assets/addreplication.png)
+
+In the pop-up window, enter the origin bucket, the remote cluster and remote bucket (in this case it's the index created earlier). Also make sure that the **XDCR Protocol** version is set to 1 and click **Save**.
```

Comment:
Setting the "XDCR Optimistic Replication Threshold" to the max value of 20971520 can have such a great impact on performance that it's probably worth including it in the basic setup instructions. Don't need to make an additional image (unless you want to); a simple mention would probably be enough.